sky/backends/cloud_vm_ray_backend.py

```diff
@@ -5358,8 +5356,6 @@ def post_teardown_cleanup(self,
                 cloud = launched_resources.cloud
                 config = global_user_state.get_cluster_yaml_dict(
                     handle.cluster_yaml)
-                ports_cleaned_up = False
```

Comment:
No need to check the cleanup, we need to remove the cluster config if no error is raised in L5402.